keyboards/mt64rgb/rules.mk

```
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no      # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+RGB_MATRIX_ENABLE = IS31FL3733 # Use RGB matrix
```

`RGB_MATRIX_ENABLE` has been split up into two definitions. `RGB_MATRIX_ENABLE` is now a `yes` or `no`, and `RGB_MATRIX_DRIVER` should be set to the driver being used.


```suggestion
RGB_MATRIX_ENABLE = yes     # Use RGB matrix
RGB_MATRIX_DRIVER = IS31FL3733
```